src/randomfourier.jl

```diff
@@ -17,38 +17,30 @@ struct RandomFourier{F, I} <: Surrogate
     inverse::I
     phases::Bool
 end
-RandomFourier(x::Bool=true) = RandomFourier(nothing, nothing, x)
-function RandomFourier(s::AbstractVector, x::Bool=true)
+
+RandomFourier(phases::Bool=true) = RandomFourier(nothing, nothing, phases)
+function RandomFourier(s::AbstractVector, phases::Bool=true)
     forward = plan_rfft(s)
     inverse = plan_irfft(forward*s, length(s))
-    return RandomFourier(forward, inverse, x)
+    return RandomFourier(forward, inverse, phases)
 end
 
-function surrogate(x::AbstractVector{T}, method::RandomFourier) where T
-    n = length(ts)
-    m = mean(x)
-    𝓕 = isnothing(method.forward) ? rfft(x .- m) : method.forward*(x .- m)
-
+function surrogate(s::AbstractVector{T}, method::RandomFourier) where T
```

Comment:
This is a good point to discuss following a convention in the code. The input timeseries should be universally represented as the same symbol in all code. `s` is a good candidate _but_: some methods require to generate the surrogate timeseries while the method runs. Thus, they have to actually assign a variable to the surrogate.

That's why I propose to use `x` to always specify input timeseries and `s` the output surrogate.

Reply:
Agreed, `x` for inputs and `s` for output surrogates are reasonable symbols.